.goreleaser.yml

```
@@ -115,7 +115,8 @@ nfpms:
     dependencies:
       - git
     # Override default /usr/local/bin destination for binaries
-    bindir: /usr/local/bin
+    # Since glab is already archived in the bin directory it would install in /usr/local/bin/bin/glab
+    bindir: /usr/local
```

Let's install it in the `/usr/bin` dir instead
```suggestion
    # Since glab is already archived in the bin directory it would install in /usr/bin/glab
    bindir: /usr
```